src/internal.h

```diff
@@ -82,6 +82,7 @@
 
 
 #include <os/object.h>
+#include "stubs_internal.h"
```

Comment:
this seems like the wrong spot for this inclusion, plus we have the shims/ directory for this type of thing already, please use shims.h and add new shims/stubs in there